src/app/actions/taskClose.ts

```
@@ -0,0 +1,11 @@
+import { logInfo } from "@/Logger";
+
+/**
+ * タスククローズを実行する
+ * @param {any} payload - ペイロード
+ * @param {string} task_id - タスクID
+ */
+export function taskClose(payload: any): GoogleAppsScript.Content.TextOutput {
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)